docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -41,13 +41,18 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    solver = sinkhorn.Sinkhorn(progress_fn=default_progress_fn)
</pre><p>I think we want to use the functional interface in the getting started tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Indeed</p>